libs/genai/langchain_google_genai/chat_models.py

```diff
@@ -1171,11 +1189,20 @@ def _prepare_request(
                 "Must specify at most one of tool_choice and tool_config, received "
                 f"both:\n\n{tool_choice=}\n\n{tool_config=}"
             )
+              
         formatted_tools = None
         if tools:
             formatted_tools = [convert_to_genai_function_declarations(tools)]
         elif functions:
             formatted_tools = [convert_to_genai_function_declarations(functions)]
+        elif self.code_execution:
```

Comment:
let's allow to pass `code_execution=True` as part of kwargs of the `invoke` method, please